deluge/ui/console/widgets/statusbars.py

```diff
@@ -11,9 +11,11 @@
 
 import deluge.common
 import deluge.component as component
-from deluge.core.preferencesmanager import DEFAULT_PREFS
 from deluge.ui.client import client
 
+# The same as the value in core.preferencesmanager.DEFAULT_PREFS['daemon_port']
```

Comment:
We don't need this comment, it's fine to assume

Reply:
ok. removed.